openmetadata-ui/src/main/resources/ui/src/components/common/DeleteWidget/DeleteWidget.tsx

```diff
@@ -42,7 +42,7 @@ const DeleteWidget = ({
     useState<typeof ENTITY_DELETE_STATE>(ENTITY_DELETE_STATE);
 
   const prepareDeleteMessage = (softDelete = false) => {
-    const softDeleteText = `Soft deleting will deactivate the ${entityName}. This will disable any discovery, read or write operations on ${entityName}`;
+    const softDeleteText = `Delete will deactivate the ${entityName}. This will disable any discovery, read or write operations on ${entityName}`;
```

Comment:
```suggestion
    const softDeleteText = `Delete will deactivate the ${entityType} ${entityName}. This will disable any discovery, read or write operations on ${entityName}`;
```